src/shadowbox/server/mocks/mocks.ts

```diff
@@ -65,3 +65,21 @@ export class FakePrometheusClient extends PrometheusClient {
     return queryResultData;
   }
 }
+
+export class FakeTunnelTimePrometheusClient extends PrometheusClient {
```

Comment:
Is it worth renaming the existing `FakePrometheusClient` to something like `FakeDataBytesTransferredPrometheusClient`?